pool.go

```diff
@@ -25,10 +26,12 @@ type ioErrConn struct {
 	// level error, e.g. a timeout, disconnect, etc... Close is automatically
 	// called on the client when it encounters a critical network error
 	lastIOErr error
+
+	ConnectTime time.Duration
```

Comment:
The struct is not exported, so it does not make sense to export this.

Reply:
I'll remove this variable.